virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -72,6 +72,9 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-17-storage"]
 === Storage
 
+//CNV-40180 Release note: API is now v1beta1
+* The VirtualMachineSnapshot API version is now v1beta1.
```

Comment:
```suggestion
* The `VirtualMachineSnapshot` API version is now v1beta1.
```
If the name of this API version is not typically in monospace font, feel free to ignore this suggestion.

Reply:
It is! Good catch; thanks!